MinhNguyen/vine_copulas.py

```diff
@@ -0,0 +1,238 @@
+import pandas as pd
+import numpy as np
+
+from itertools import combinations
+from tqdm import tqdm
+from statsmodels.distributions.empirical_distribution import ECDF
+
+from stat_utils import SpearmanExt
+
+
+class PartnerSelector():
+    """An implementation of the Partner Selection methods as described in 
```

Comment:
Nice docstring. I like the attributes section but wonder if it will work with Sphinx.